### DISCUSSION TITLE
make water edge water instead of dirt

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#18128] [Feature requests]
Could somebody please make terrain edge for water that is actually water? right now in dozens of contests for years, we have to use abstract walls or other hacky things for water along edges of parks. It would be so much better if devs could program in some kind of terrain edge that uses the same code as the water surface...

**Comment by Broxzier** [comment]
We'd need some good sprites for that then. Water without walls on the edge of the map looks worse than with walls.

![Crazy Castle 2022-09-28 21-25-43](https://user-images.githubusercontent.com/9705046/192871158-61477918-40fb-46a8-817e-ab8b95dc984a.png)

**Comment by jjpb44** [comment]
Any suggestions for the water edge sprites?

**Comment by 733737** [comment]
![00007](https://user-images.githubusercontent.com/94884086/193483562-970d97ac-f81c-464c-9a7c-f4d282d1f914.png)
![00008](https://user-images.githubusercontent.com/94884086/193483563-103b89a6-c4e1-4aa0-af7f-6fd45daa3e65.png)
![00009](https://user-images.githubusercontent.com/94884086/193483564-07e4626a-23bf-4c4d-821b-840a268ec9b9.png)
![00065](https://user-images.githubusercontent.com/94884086/193483594-abe123c3-7ecd-4aea-94de-30b573d36bfb.png)
![00067](https://user-images.githubusercontent.com/94884086/193483595-1e9f4b5b-b5a9-4b27-9291-4bb2ee0dc76e.png)
![00068](https://user-images.githubusercontent.com/94884086/193483596-d6ae599a-3b28-4f94-bc0f-77a3afb56f9a.png)
![00069](https://user-images.githubusercontent.com/94884086/193483597-59ffd6c7-fae2-4581-85e4-15a3e7ef2724.png)
![00071](https://user-images.githubusercontent.com/94884086/193483598-2e447134-b42d-4609-be6a-b798d33b8b3d.png)
![00073](https://user-images.githubusercontent.com/94884086/193483599-233f0f7a-5d86-4839-989a-2d93011a4c24.png)
![00075](https://user-images.githubusercontent.com/94884086/193483600-609fe31a-406e-4268-bf53-a0ee3616f30e.png)
![00081](https://user-images.githubusercontent.com/94884086/193483601-f799a70a-1ce6-44df-8e78-70a379578063.png)
![00049](https://user-images.githubusercontent.com/94884086/193483602-3d65f2ee-2a1e-4b18-9dcd-732bef6b48cc.png)
![00061](https://user-images.githubusercontent.com/94884086/193483604-f3ccf76f-6615-45d5-9683-b5b596a21a9c.png)
![00063](https://user-images.githubusercontent.com/94884086/193483605-3b3a31e0-4bd5-4163-a9d7-f1b5c98d2e74.png)
![00010](https://user-images.githubusercontent.com/94884086/193483565-48652826-6314-4199-93da-006bfb980728.png)
![00011](https://user-images.githubusercontent.com/94884086/193483566-4c4eba45-1df2-42b6-8513-e8979ce6cc2f.png)
![00012](https://user-images.githubusercontent.com/94884086/193483567-c9861078-fd55-4b5c-93f0-fe72ccf46490.png)
![00013](https://user-images.githubusercontent.com/94884086/193483568-df91553b-3f7b-44be-afff-cb347147c1b9.png)
![00014](https://user-images.githubusercontent.com/94884086/193483569-896540cb-3a6b-48c9-8f23-01687167cb55.png)
![00015](https://user-images.githubusercontent.com/94884086/193483570-65b9707f-211a-4c4f-b84d-711d2f48d367.png)
![00016](https://user-images.githubusercontent.com/94884086/193483571-c55a663b-e4ee-4c6b-89e2-a4c89b705322.png)
![00017](https://user-images.githubusercontent.com/94884086/193483572-f5f4082a-0396-46e1-bd98-07124aaa75fb.png)
![00018](https://user-images.githubusercontent.com/94884086/193483573-da870094-653a-4f5a-8aff-e9ed76622f8d.png)
![00019](https://user-images.githubusercontent.com/94884086/193483574-e28436cc-ace3-4d1d-af17-4f10a42ef5a0.png)
![00020](https://user-images.githubusercontent.com/94884086/193483575-0098e3b1-599a-4e11-8c7e-e3ecedd95518.png)
![00000](https://user-images.githubusercontent.com/94884086/193483576-b97f98d0-b6f9-4adc-b61e-ef6d92c849df.png)
![00001](https://user-images.githubusercontent.com/94884086/193483577-89075e49-3476-402a-ba7f-bafc8d1f6139.png)
![00002](https://user-images.githubusercontent.com/94884086/193483579-463abc92-b884-4ec3-94e8-8f9c9caebdc2.png)
![00003](https://user-images.githubusercontent.com/94884086/193483580-88110432-77e5-496f-8eda-b3ce25131685.png)
![00004](https://user-images.githubusercontent.com/94884086/193483581-68c29e1e-9990-4634-9853-91a455fb6fe5.png)
![00005](https://user-images.githubusercontent.com/94884086/193483582-4d2bd299-98bf-4816-af06-338e9c3e8cee.png)
![00006](https://user-images.githubusercontent.com/94884086/193483583-35f3ff6e-9ee0-4107-a1ab-6811b75a6725.png)




struggling to get my object to work

**Reply by IntelOrca** [reply]
If you upload your object then we can tell you what the problem is.

**Comment by 733737** [comment]
[73.terrain_edge.water.zip](https://github.com/OpenRCT2/OpenRCT2/files/9697426/73.terrain_edge.water.zip)
![Github Park 2022-10-03 08-22-22](https://user-images.githubusercontent.com/94884086/193576017-a5c8168f-dbd4-4670-b695-d3da3214a348.png)
should I change format from rle to raw? and does terrain edge have to be 8 bit?
